include/calculate_fusion.h

```diff
@@ -0,0 +1,119 @@
+/**
+ * @file calculate_fusion.h
+ *
+ * @brief A collection of functions related to the calculation of fused output.
+ *
+ * @author Karthik Dilliraj - karthikdilliraj@cmail.carleton.ca - Carleton University
+ * @author Nhat Hieu Le - nhathieule@cmail.carleton.ca - Carleton University
+ * @author Jason Miller - jasonmiller@cmail.carleton.ca - Carleton University
+ */
+
+#ifndef CALCULATE_FUSION_H
+#define CALCULATE_FUSION_H
+
+#include "linked_list.h"
+
+/**
+ * @brief Structure to store eigen_systems
+ *
+ * @details Structure eigen_systems stores pointer to double of eigen_value and pointer to pointer to double of eigen_vector
+ */
+struct eigen_systems
+{
+    double *eigen_value;
+    double **eigen_vector;
+};
+
+/**
+ * @brief Calculate Support degree matrix D for the set of sensor values
+ *
```

Comment:
You need to include a detailed description in addition to the brief description for each function.

Reply:
Detailed descriptions are added